### PR TITLE
Create Vaccine Requests - Set Applicant File Nullable 

### PR DESCRIPTION
**Pull request by rindibudiaramdhan**
## Evidence
title: Create Vaccine Requests - Set Applicant File Nullable
project: Pikobar Logistik
participants: @budiaramdhanrindi @fimz17 @rizkyrmsyah @firmanalamsyah580

### CODE REVIEW

**Review by firmanJS — APPROVED** [review]

**Review by rizkyrmsyah — APPROVED** [review]

**Review by ayocodingit — APPROVED** [review]

File: app/VaccineRequest.php [hunk 1]
```diff
@@ -27,12 +27,14 @@ class VaccineRequest extends Model
 
     public function getLetterFileUrlAttribute($value)
     {
-        return config('aws.url') . $value;
+        $awsUrl = config('aws.url');
+        return $value ? $awsUrl . $value : "";
```
Comment (rachadiannovansyah) [inline app/VaccineRequest.php:31]: we're love ternary's :heart:

**Review by rachadiannovansyah — APPROVED** [review]

File: app/VaccineRequest.php (hunk 1 above)
Comment (rindibudiaramdhan) [reply app/VaccineRequest.php:31]: yah novan ketinggalan evidence
🙏

Comment (setiadijoe) [reply app/VaccineRequest.php:31]: I want to ask, why is still need checking when get the value? why don't just do it inside config package? So when we use variable from config, it has already clear and we didn't need to check again?

Comment (setiadijoe) [reply app/VaccineRequest.php:31]: wah telat uy. wkwkwk
ya sudahlah

**Pull request merged**